code-challenges.test.js

```diff
@@ -106,11 +141,14 @@ describe('newDoublearr', () => {
   })
 })
 
+//create function
+//2. takes 2 arguments arr1,arr2
 
 const newDoublearr = (arr1,arr2)=>{
```

Comment:
Excellent work on this problem!  Would love to know why you decided to use function notation inside your filter?